src/components/Accordion/Accordion.Skeleton.js

```diff
@@ -10,18 +10,25 @@ import React from 'react';
 import Icon from '../Icon';
 import SkeletonText from '../SkeletonText';
 import { iconChevronRight } from 'carbon-icons';
+// TODO: import { ChevronRight } from '@carbon/icons-react';
```

Comment:
I think this default entrypoint isn't going to happen because we have >2000 icon exports now 😬 

Ended up removing it from the icon package upstream because of babel having issues processing a file that large/with that many exports.

I bet we could get away with the `es` directory direct import for our components 👍

Reply:
gotcha, will prob make a new PR to remove these TODOs then